test/functional/api/vim_spec.lua

```diff
@@ -3733,6 +3733,12 @@ describe('API', function()
       eq("", meths.cmd({ cmd = "Foo", bang = false }, { output = true }))
     end)
     it('works with modifiers', function()
+      -- with :silent output is still captured
+      eq('1',
+         meths.cmd({ cmd = 'echomsg', args = { '1' }, mods = { silent = true } },
+                   { output = true }))
+      -- with :silent message isn't added to message history
+      eq('', meths.cmd({ cmd = 'messages' }, { output = true }))
```

Comment:
the comment says "with :silent" but silent is not specified in the code. Is that intended?

Reply:
This refers the the previous `echomsg` command